packages/core/src/node/process/raw-process.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject } from 'inversify';
+import { ILogger } from '../../common/logger';
+import { Process } from './process';
+import * as child from 'child_process';
+import * as stream from 'stream';
+
+export const RawProcessOptions = Symbol("RawProcessOptions");
+export interface RawProcessOptions {
+    command: string,
+    args?: string[],
+    options?: object
+}
+
+export const RawProcessFactory = Symbol("RawProcessFactory");
+export type RawProcessFactory = (options: RawProcessOptions) => RawProcess;
+
+@injectable()
+export class RawProcess extends Process {
+
+    readonly type: 'Raw' | 'Terminal' = 'Raw';
```

Comment:
Well I just wanted to make it clear what the possible types were, I don't think it hurts?
I could have made an enum too